Source/Src_nd/diag.F90

```diff
@@ -1,56 +1,58 @@
+! diag module for MAESTROeX. 
+! currently, there are 3 output files:
+!
+!   diag_enuc.out:
+!          peak nuc energy generation rate (erg / g / s)
+!          x/y/z location of peak enuc
+!          velocity components at location of peak enuc (including vr)
+!          radius of peak enuc
+!          total nuclear energy release (erg / s)
+!
+!   diag_temp.out:          
+!          peak temperature
+!          x/y/z location of peak temperature
+!          velocity components at location of peak temperature (including vr)
+!          radius of peak temperature
+!          central temperature
+!
+!   diag_vel.out:
+!          peak total velocity
+!          peak Mach number
+!          total kinetic energy
+!          gravitational potential energy
+!          total internal energy
+!          velocity components at the center of the star
+!
 module diag_module
-  ! diag module for MAESTROeX.
-  ! currently, there are 3 output files:
-  !
-  ! diag_enuc.out:
-  ! peak nuc energy generation rate (erg / g / s)
-  ! x/y/z location of peak enuc
-  ! velocity components at location of peak enuc (including vr)
-  ! radius of peak enuc
-  ! total nuclear energy release (erg / s)
-  !
-  ! diag_temp.out:
-  ! peak temperature
-  ! x/y/z location of peak temperature
-  ! velocity components at location of peak temperature (including vr)
-  ! radius of peak temperature
-  ! central temperature
-  !
-  ! diag_vel.out:
-  ! peak total velocity
-  ! peak Mach number
-  ! total kinetic energy
-  ! gravitational potential energy
-  ! total internal energy
-  ! velocity components at the center of the star
 
   use meth_params_module, only: rho_comp, spec_comp, temp_comp, prob_lo, &
-       sponge_start_factor, sponge_center_density, base_cutoff_density
+       sponge_start_factor, sponge_center_density, base_cutoff_density, &
+       grav_const
   use network, only: nspec
   use base_state_geometry_module, only:  max_radial_level, nr_fine, center
   use eos_module
   use eos_type_module
   use amrex_constants_module
   use fundamental_constants_module, only: Gconst
-
+ 
   implicit none
 
   private
 
 contains
 
-  !xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
   subroutine diag(lev, lo, hi, &
-                  scal, s_lo, s_hi, nc_s, &
-                  rho_Hnuc, hn_lo, hn_hi, &
-                  rho_Hext, he_lo, he_hi, &
-                  rho0, p0, &
-                  u, u_lo, u_hi, &
-                  w0, dx, &
-                  Mach_max,temp_max,enuc_max,Hext_max, &
-                  mask,     m_lo, m_hi, use_mask) &
-                  bind(C, name="diag")
-    ! Binds to C function ``diag``
+                       scal, s_lo, s_hi, nc_s, &
+                       rho_Hnuc, hn_lo, hn_hi, &
+                       rho_Hext, he_lo, he_hi, &
+                       u, u_lo, u_hi, &
+                       w0, dx, &
+                       T_max, coord_Tmax, vel_Tmax, &
```

Comment:
we tend to use `temp_max` rather than `T_max` as Fortran is case insensitive and so using `T` to denote temperature-related quantities can cause confusing with time-relate quantities

Reply:
I've done this throughout the fortran.

There is quite a lot of "capital T = temperature" in the cpp still, do you want that changed?